src/skin/legacyskinparser.cpp

```diff
@@ -488,6 +489,10 @@ QList<QWidget*> LegacySkinParser::parseNode(QDomElement node) {
         m_pContext->updateVariable(node);
     } else if (nodeName == "Template") {
         result = parseTemplate(node);
+    } else if (nodeName == "DefineSingleton") {
+        parseSingletonDefinition(node);
+    } else if (nodeName == "Singleton") {
+        result = wrapWidget(parseSingletonContainer(node));
```

Comment:
```Singleton``` is particularly confusing because ```PushButton``` creates a push-button so you'd expect that ```Singleton``` creates a singleton. So I'd say put a View or Container on it or something to make it clear it's the thing that is showing the singleton. 

I agree with Daniel about the verbs #bikeshed.